src/index.html

```diff
@@ -2,12 +2,503 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    >
+    <title>BIKE</title>
     <link rel="stylesheet" href="./styles/main.scss">
-    <script src="scripts/main.js" defer></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page">
+    <header class="header">
+      <div class="container">
+        <div class="top-action">
+          <div class="logo"></div>
+          <a
+            href="tel: +1 234 5555-55-55"
+            class="icon icon--phone"
+          >
+          </a>
```

Comment:
this number is a bit offcentered
![image](https://user-images.githubusercontent.com/37161315/170644792-6477734a-8d29-4013-bc5d-c0194b84f08c.png)